content/en/synthetics/guide/app-that-requires-login.md

```diff
@@ -2,16 +2,18 @@
 title: Running tests on an application that requires authentication
 kind: guide
 further_reading:
-    - link: 'synthetics/browser_tests'
-      tag: 'Documentation'
-      text: 'Configure a Browser Test'
-    - link: '/synthetics/browser_tests/actions'
-      tag: 'Documentation'
-      text: 'Create Browser Test Steps'
-    - link: 'https://www.datadoghq.com/blog/test-creation-best-practices/'
-      tag: 'Blog'
-      text: 'Best practices for creating end-to-end tests'
-
+  - link: 'https://www.datadoghq.com/blog/mfa-synthetic-testing-datadog/'
+    tag: 'Blog'
+    text: 'Introducing multi-factor authentication in Datadog Synthetic tests'
+  - link: 'synthetics/browser_tests'
+    tag: 'Documentation'
+    text: 'Configure a Browser Test'
+  - link: '/synthetics/browser_tests/actions'
+    tag: 'Documentation'
+    text: 'Create Browser Test Steps'
```

Comment:
```suggestion
    text: 'Configure a browser test'
  - link: '/synthetics/browser_tests/actions'
    tag: 'Documentation'
    text: 'Create browser test steps'
```

Reply:
Personally, I'd rather the text be the page title (title case). I will update.

Reply:
```suggestion
    text: 'Browser Test Steps'
```